### PR TITLE
Add Save/Load

### PR DESCRIPTION
**Pull request by DexOfTheWild**
Should save anytime you complete editing a new photo
@bblia @terraoldham

### CODE REVIEW

File: nailed-it/New Group/TryItOnViewController.swift [hunk 1]
```diff
@@ -20,10 +19,42 @@ class TryItOnViewController: UIViewController, UIImagePickerControllerDelegate,
...
+                applyImageMask()
+            }
+        } else {
+            selectColorButton.isEnabled = false
+        }
+    }
+    
+    func applyImageMask() {
+        // adjust the size of image view to make it fit the image size and put it in the center of screen
+        var x:CGFloat, y:CGFloat, width:CGFloat, height:CGFloat
```
Comment (bblia) [inline nailed-it/New Group/TryItOnViewController.swift]: why are we adding this stuff back in?

Comment (DexOfTheWild) [reply nailed-it/New Group/TryItOnViewController.swift]: oh whoops, I think my rebase messed up and I thought it was still around.

Comment (bblia) [reply nailed-it/New Group/TryItOnViewController.swift]: https://github.com/tolzar/nailed-it/pull/18/files#diff-f17fb3c95c0e84f1802724b850704306L59

**Comment by bblia** [comment]
lgtm

**Pull request merged**